src/main/java/edu/kit/hci/soli/domain/NonOidcUser.java

```diff
@@ -0,0 +1,11 @@
+package edu.kit.hci.soli.domain;
+
+import jakarta.persistence.Entity;
+import lombok.Data;
+
+@Entity
+@Data
+public class NonOidcUser extends User{
+
+    private String password;
```

Comment:
Why does a NonOidcUser have a password? Is this a guest account?

Reply:
Yes, NonOidcUser is a guest account and thus have no password